Rubberduck.Parsing/Grammar/VBAParser.g4

```diff
@@ -486,6 +486,10 @@ selectEndValue : expression;
 
 setStmt : SET whiteSpace lExpression whiteSpace? EQ whiteSpace? expression;
 
+stepStmt :
+	STEP whiteSpace expression
+;
```

Comment:
I'll need to pull those changes over into the ANTLR Upgrade as well, remdinder to myself to do that